packages/palette/src/palette.spec.ts

```diff
@@ -0,0 +1,20 @@
+import { uiColors } from '.';
+
+interface HueRange {
+  dark3: string;
+  dark2?: string;
+  dark1?: string;
+  base: string;
+  light1?: string;
+  light2?: string;
+  light3: string;
+}
+
+const allColors: Array<string | HueRange> = Object.values(uiColors);
```

Comment:
If we export one named object with `as const` rather than have a bunch of named exports from `uiColors`, can the type of `allColors` be inferred here?

Reply:
As mentioned in the other comment, this line is explicitly typed on purpose, to try and make sure that we're exporting a consistent shape from `uiColors`, while still allowing `as const` to do its thing.